edb/server/bootstrap.py

```diff
@@ -836,6 +836,8 @@ def prepare_patch(
             configspec=config.spec_to_json(config_spec).encode('utf-8'),
         ))
 
+        support_view_commands.generate(subblock)
```

Comment:
During a refactor I accidentally deleted this line that actually recreates the views